New policy certificate-otp for R2W in nethgui (#61)
NethServer/nethserver-openvpn: root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_VPN.rst

    * Certificate: choose when creating a net2net configuration
    * User, password and certificate: this is the most secure combination.
      The user must be a system user.
    * User, one time password and certificate: This policy provides a multi

>       factor of Authentication, the user must install an OTP application on his 
>       mobile phone, the code changes every 30 seconds. The user must be a system-user. 

Reviewer: ```suggestion
      factor of Authentication, users must enable 2FA from new Server Manager and  they will be required 
      to provide a temporary verification code (OTP) generated by an application running on their smartphone. 
      The user must be a system-user. 
```

Author: done

    * User, password and certificate: this is the most secure combination.
      The user must be a system user.
    * User, one time password and certificate: This policy provides a multi

>       factor of Authentication, users must enable 2FA from new Server Manager 

Reviewer: ```suggestion
      factor of authentication, users must enable 2FA from new Server Manager 
```

Author: done

>       and  they will be required to provide a temporary verification code (OTP) 

Reviewer: ```suggestion
      and they will be required to provide a temporary verification code (OTP) 
```

Author: done
